overdrive.py

```diff
@@ -619,7 +619,7 @@ def book_info_to_metadata(cls, book):
             grade_level = grade_level_info.get('value')
             subjects.append(
                 SubjectData(type=Subject.GRADE_LEVEL, identifier=grade_level,
-                            weight=1)
+                            weight=100)
```

Comment:
Overdrive grade levels turn out to be about as reliable as other Overdrive data.